js/renovation/viz/core/__tests__/tooltip.test.tsx

```diff
@@ -76,7 +76,7 @@ describe('Render', () => {
     visible: true,
   };
 
-  const props = {
+  const viewModel = {
```

Comment:
Why do you rename it?

Reply:
Because it's not 'props' - it's viewModel

Reply:
I just want to make a consistent code in a whole project. If all teams use the `props` word to declare data we should do the same.

Reply:
const props = {
      props: {..}
}
Looking so weird...

Reply:
Fixed